codecov_cli/services/staticanalysis/analyzers/python/node_wrappers.py

```diff
@@ -10,6 +10,42 @@ def visit(self, node):
         for c in node.children:
             self.visit(c)
 
+    def _is_function_docstring(self, node):
+        """Skips docstrings for funtions, such as this one.
+        Pytest doesn't include them in the report, so I don't think we should either,
+        at least for now.
+        """
+        # Docstrings for a module are OK - they show up in pytest result
+        # Docstrings for a class are OK - they show up in pytest result
+        # Docstrings for functions are NOT OK - they DONT show up in pytest result
+        # Check if it's docstring
+        has_single_child = len(node.children) == 1
+        only_child_is_string = node.children[0].type == "string"
+        # Check if is the first line of a function
+        parent_is_block = node.parent.type == "block"
+        first_exp_in_block = node.prev_named_sibling is None
+        is_in_function_context = (
+            node.parent.parent.type == "function_definition"
+            if parent_is_block and node.parent is not None
+            else False
+        )
```

Comment:
i had to read this a couple times to follow it. the `==` expression in the if-true clause of the ternary threw me for a loop, and i think the `node.parent is not None` check is implicitly covered in `parent_is_block` (it would error there if `parent` doesn't exist, right?)

i think you can one-line this:
```
is_in_function_context = parent_is_block and node.parent.parent.type == "function_definition"
```

Reply:
ah you are correct. Thanks.